karapace/karapace.py

```diff
@@ -67,10 +66,6 @@ def close(self):
         self.producer.close()
         self.producer = None
 
-    @staticmethod
-    def read_config(config_path):
-        return read_config(config_path)
```

Comment:
The places that need to read the config are just calling `read_config` directly.